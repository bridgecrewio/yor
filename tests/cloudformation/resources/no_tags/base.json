{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Resources": {
    "S3Bucket": {
      "Type": "AWS::S3::Bucket",
      "Properties": {
        "Tags": [
          {
            "Key": "yor_trace",
            "Value": "f318146e-b3a1-4d8c-85d2-5895f744af76"
          }
        ],
        "AccessControl": "PublicRead",
        "WebsiteConfiguration": {
          "IndexDocument": "index.html",
          "ErrorDocument": "error.html"
        }
      },
      "DeletionPolicy": "Retain"
    }
  }
}