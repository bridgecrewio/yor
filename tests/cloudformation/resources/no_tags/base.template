AWSTemplateFormatVersion: 2010-09-09
Resources:
  S3Bucket:
    Type: AWS::S3::Bucket
    Properties:
      AccessControl: PublicRead
      WebsiteConfiguration:
        IndexDocument: index.html
        ErrorDocument: error.html
    DeletionPolicy: Retain
