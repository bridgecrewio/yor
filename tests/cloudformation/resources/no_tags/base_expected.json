{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Resources": {
    "S3Bucket": {
      "Type": "AWS::S3::Bucket",
      "Properties": {
        "Tags": [
          {
            "Key": "new_tag",
            "Value": "new_value"
          },
          {
            "Key": "another_tag",
            "Value": "another_val"
          }
        ],
        "AccessControl": "PublicRead",
        "WebsiteConfiguration": {
          "IndexDocument": "index.html",
          "ErrorDocument": "error.html"
        }
      },
      "DeletionPolicy": "Retain"
    }
  }
}
